Plugins/Geant4/include/Acts/Plugins/Geant4/Geant4PhysicalVolumeSelectors.hpp

```diff
@@ -0,0 +1,82 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/Delegate.hpp"
+
+#include <string>
+#include <vector>
+
+#include "G4VPhysicalVolume.hh"
+
+namespace Acts {
+
+class IGeant4PhysicalVolumeSelector {
+ public:
+  virtual ~IGeant4PhysicalVolumeSelector(){};
```

Comment:
Is it on purpose that the Base class does not have the `bool select(...) const` interface?

Reply:
I just tried to keep it consistent.